src/lib/tokenUtils.ts

```diff
@@ -34,6 +34,17 @@ export function pluralizeVote(count: BigInt) {
     .join("")} votes`;
 }
 
+export function pluralizeSnapshotVote(count: bigint) {
```

Comment:
Can this be abstracted to just pluralizeVote?

Reply:
The problem is we already have a `pluralizeVote` in this file, but it's expecting a different unit so it was breaking for snapshot votes. Hence the snapshot specific votes. I agree it would be nice to have a single function but would probably require a change more upstream where the data is indexing.